docs/tutorial.rst

```diff
@@ -1256,7 +1256,7 @@ as aliases, by wrapping them in ``@()``.  For example:
 
     >>> @(_banana)
     'My spoon is tooo big!'
-    >>> echo "hello" | @(lambda args, stdin=None: stdin.strip() + args[0]) world
+    >>> echo "hello" | @(lambda args, stdin=None: ' '.join([stdin.read().strip(), *args]) + '\n') world
```

Comment:
I think that since this is a lambda applied in place, generality is not needed here.  Would you mind simplifying it please? Thanks again for catching this.